Cargo.toml

```diff
@@ -25,7 +25,7 @@ crossbeam-channel = "0.5.14"
 crossbeam-queue = "0.3.12"
 dashmap = "6.0.1"
 datafusion = "44.0.0"
-deltalake = "0.24.0"
+deltalake = { git = "https://github.com/ModelarData/delta-rs", rev = "1c5924931ba105305b0bae86795b639de43f9be3" }
```

Comment:
Should this be changed to a numbered version before merging?

Reply:
Our fork of `deltalake` is needed until the next version is released due to #289. I have added a comment to explain this.